openedx2zim/scraper.py

```diff
@@ -247,6 +249,10 @@ def prepare_mooc_data(self):
             + "&depth=all&requested_fields=graded,format,student_view_multi_device&student_view_data=video,discussion&block_counts=video,discussion,problem&nav_depth=3"
         )
         self.course_xblocks = xblocks_data["blocks"]
+        if self.debug:
+            with tempfile.NamedTemporaryFile(dir=self.build_dir.joinpath("logs"), suffix=".json", mode="wt", delete=False) as fp:
```

Comment:
Same

Reply:
This is not an error. And this is pretty big data, so I would prefer not to log it to stdout everytime we run in debug (especially since this is the default ...). This is mostly useful only for dev on a local machine, as a convenience to not call the API on your own since it is complex due to authorization. I would prefer to keep this in a temp file.

Reply:
OK I get it ; is this frequent and large? My point being: should we create it on a different option than `debug` or not? `debug` is not default in the scraper (at least should not be) but indeed in the zimfarm we always enable it

Reply:
This is not frequent, this is one big document we get only once from the API and contains all xblocks with their data. On a given course we have about 1K xblock, and each xblock is a dict which is about 100 bytes, so I would say this gives about 100K as a whole.

Having easy access to it in dev is useful to check the xblock IDs, URLs or stuff like it.

In prod, this is almost useless, even in case of failure (and this is almost the first thing the scraper retrieve, so we can always restart the scraper locally to get it if needed).

From my PoV, there is little value to add a specific parameter, as you said debug should not be active in prod on a nominal case and writing one file in a temp folder when debugging is not a problem even if we do not use it.